packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -561,6 +561,7 @@ const Cart = () => {
             />
 
             <UnavailableProductConfirmationModal
+                productItems={basket?.productItems}
                 productIds={productIds.split(',')}
```

Comment:
Similarly, there's another file that needs to swap `productIds` with `productItems` prop: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/d76289c1a30669c56b7a6cfaf8466440df3d33fb/packages/template-retail-react-app/app/pages/account/wishlist/index.jsx#L226-L227